templates/html/signup_clinic_confirmation.html

```diff
@@ -53,6 +53,9 @@
                       <p class="content-width" style="font-size:14px;line-height:1.5;margin:0;margin-bottom:10px;margin-left:auto;margin-right:auto;max-width:400px;">
                         {{.EmailExpirationDays}}
                       </p>
+                      <p class="content-width" style="font-size:14px;line-height:1.5;margin:0;margin-bottom:10px;margin-left:auto;margin-right:auto;max-width:400px;">
+                        {{.SignupClinicDeletionWarning}}
+                    </p>
```

Comment:
should we highlight it ? 
```suggestion
                      <p class="content-width" style="font-size:14px;font-weight:bold;line-height:1.5;margin:0;margin-bottom:10px;margin-left:auto;margin-right:auto;max-width:400px;">
                        {{.SignupClinicDeletionWarning}}
                    </p>
```